doc/source/ray-air/examples/convert_existing_tf_code_to_ray_air.ipynb

```diff
@@ -0,0 +1,1070 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "42e4a113",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mnist = tf.keras.datasets.mnist\n",
+    "\n",
+    "(x_train, y_train), (x_test, y_test) = mnist.load_data()\n",
+    "x_train, x_test = x_train / 255.0, x_test / 255.0\n",
+    "\n",
+    "train_ds = tf.data.Dataset.from_tensor_slices(\n",
+    "    (x_train, y_train)).shuffle(len(x_train)).batch(32)\n",
+    "\n",
+    "test_ds = tf.data.Dataset.from_tensor_slices((x_test, y_test)).batch(32)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74677ecc",
+   "metadata": {},
+   "source": [
+    "Next, we define the model architecture of the neural network. We wrap the model definition inside a function for easy reuse later."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "515470bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def build_model() -> tf.keras.Model:\n",
+    "    return tf.keras.Sequential(\n",
+    "        [\n",
+    "            tf.keras.layers.InputLayer(input_shape=(28, 28)),\n",
+    "            tf.keras.layers.Flatten(),\n",
+    "            tf.keras.layers.Dense(128, activation=\"relu\"),\n",
+    "            tf.keras.layers.Dropout(0.2),\n",
+    "            tf.keras.layers.Dense(10),\n",
+    "        ]\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93196482",
+   "metadata": {},
+   "source": [
+    "Next, initialize the model, loss, optimizer, and define some metrics that we want to track during training.\n",
+    "\n",
+    "We recommend using the Keras `Model.fit` API, as it simplifies distributing your training with `tf.distribute` and Ray AIR. Compile your model with a loss function and optimizer, then run `model.fit(train_ds)`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "8ba828bd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "loss_object = tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True)\n",
+    "optimizer = tf.keras.optimizers.Adam()\n",
+    "\n",
+    "model = build_model()\n",
+    "model.compile(\n",
+    "    optimizer=optimizer,\n",
+    "    loss=loss_object,\n",
+    "    metrics=[\"accuracy\"],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "84c8a531",
+   "metadata": {},
+   "source": [
+    "Lastly, we have the training loop, which trains the model for some number of epochs. Each epoch will loop through the entire training dataset and perform gradient descent steps.\n",
+    "\n",
+    "One slight difference between the Tensorflow tutorial and this training loop is that we choose to train one epoch at a time rather than specifying multiple epochs through `model.fit(..., epochs=epochs)`. This will allow us to perform evaluation on the test dataset per epoch, as well as enable per epoch checkpointing and reporting."
```

Comment:
can we keep the standard approach of `model.fit()` and show how to use the Ray AIR KerasCallback instead?

we could also show both approaches